benchmark/benchmark_utils.py

```diff
@@ -68,26 +71,28 @@ def benchmark(self, n):
         """
 
 
-def create_qnode(qfunc, device, mutable=True, interface="autograd"):
+def create_qnode(qfunc, device, mutable=True, interface="autograd", qnode_type="QNode"):
     """Utility function for creating a quantum node.
 
     Takes care of the backwards compatibility of the benchmarks.
 
-    Implicitly uses the parameter-shift method for computing Jacobians.
+    By default, uses the parameter-shift method for computing Jacobians.
```

Comment:
Unless the device supports passthru, in which case it will default to backprop 🤔 The default here will default to the 'best' ordering that is currently defined in `qnodes/decorator.py`.

Reply:
Not my docstring originally, but I will fix it